### PR TITLE
[13.0][MIG] product_secondary_unit

### PR DESCRIPTION
**Pull request by ps-tubtim**

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: product_secondary_unit/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Product Secondary Unit",
+    "summary": "Set a secondary unit per product",
+    "version": "13.0.1.0.0",
+    "development_status": "Beta",
```
Comment (sergio-teruel) [inline product_secondary_unit/__manifest__.py]: This module could be in "Production/Stable", If you do this change you also must be add a mantainers key...

File: product_secondary_unit/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "category": "Product",
+    "website": "https://github.com/OCA/product-attribute",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": ["product"],
+    "data": ["security/ir.model.access.csv", "views/product_views.xml"],
```
Comment (sergio-teruel) [inline product_secondary_unit/__manifest__.py:15]: Put every element in diferent lines.. like as v11.0
https://github.com/OCA/product-attribute/blob/11.0/product_secondary_unit/__manifest__.py#L17

File: product_secondary_unit/models/product_second_unit.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        comodel_name="product.template", string="Product Template", required=True
+    )
+    uom_id = fields.Many2one(
+        comodel_name="uom.uom",
+        string="Secondary Unit of Measure",
+        required=True,
+        help="Default Secondary Unit of Measure.",
+    )
+    factor = fields.Float(
+        string="Secondary Unit Factor", default=1.0, digits=0, required=True
```
Comment (sergio-teruel) [inline product_secondary_unit/models/product_second_unit.py:22]: Can you maintain the format from last  version??

File: product_secondary_unit/tests/test_product_second_unit.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests import SavepointCase
+
+
+class TestProductSecondaryUnit(SavepointCase):
+    at_install = False
```
Comment (sergio-teruel) [inline product_secondary_unit/tests/test_product_second_unit.py]: Please use tagged (@tagged('post_install', '-at_install'))

File: product_secondary_unit/tests/test_product_second_unit.py [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.product_uom_kg = cls.env.ref("uom.product_uom_kgm")
+        cls.product_uom_unit = cls.env.ref("uom.product_uom_unit")
+        cls.product = cls.env["product.template"].create(
+            {
+                "name": "test",
+                "uom_id": cls.product_uom_kg.id,
+                "uom_po_id": cls.product_uom_kg.id,
+                "secondary_uom_ids": [
```
Comment (sergio-teruel) [inline product_secondary_unit/tests/test_product_second_unit.py:18]: Maintain format like as previous version...

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

File: product_secondary_unit/__manifest__.py (hunk 2 above)
Comment (pedrobaeza) [reply product_secondary_unit/__manifest__.py:15]: This is done automatically by black

File: product_secondary_unit/tests/test_product_second_unit.py (hunk 5 above)
Comment (pedrobaeza) [reply product_secondary_unit/tests/test_product_second_unit.py:18]: This is done automatically by black

File: product_secondary_unit/models/product_second_unit.py (hunk 3 above)
Comment (pedrobaeza) [reply product_secondary_unit/models/product_second_unit.py:22]: This is done automatically by black

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**